orchagent/nexthopkey.h

```diff
@@ -15,9 +17,24 @@ struct NextHopKey
     string              alias;          // incoming interface alias
     uint32_t            vni;            // Encap VNI overlay nexthop
     MacAddress          mac_address;    // Overlay Nexthop MAC.
+    LabelStack          label_stack;    // MPLS label stack
 
     NextHopKey() = default;
-    NextHopKey(const std::string &ipstr, const std::string &alias) : ip_address(ipstr), alias(alias), vni(0), mac_address() {}
+    NextHopKey(const std::string &str, const std::string &alias) : alias(alias), vni(0), mac_address()
```

Comment:
Exercise all the constructors in the unit test. Add get_apis() to verify the parsed results.